code-studio/src/js/initApp/initApp.js

```diff
@@ -6,6 +6,11 @@ var chrome34Fix = require('./chrome34Fix');
 var loadApp = require('./loadApp');
 var project = require('./project');
 
+function isMobile() {
```

Comment:
Yeah, that's where I stole it from. Our pipelines still don't have great support for sharing code between apps and code-studio